data/transport missions.txt

```diff
@@ -1412,6 +1415,139 @@ mission "Hallucination"
+	name "Shady passenger transport"
+	description "Discreetly transport a passenger to <destination>. If you are caught with him, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" "core" "urban"
+	destination
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+		distance 2 3
+	passengers 1
+	illegal 75000 `Your passenger turns out to be a small-time wanted criminal. As he is hauled off in chains, you vehemently protest that you knew nothing of his shady history. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+
+	to offer
+		random < 10
+		"rejected illegal jobs" < 3
+
+	on offer
+		dialog `While having a drink at one of one of the spaceport's bars, a somewhat scruffy-looking man in a dark cloak pulls you aside and offers <payment> for discreet, no-questions-asked passage to <destination>, which is quite close by. This could be a quick and lucrative job. (It also sounds illegal, so you'll need to avoid getting scanned or landing on planets with high security.)`
+
+	on accept
+		"rejected illegal jobs" --
+
+	on decline
+		"rejected illegal jobs" ++
+
+	on complete
+		"getaway driver" ++
+		payment 50000
+		dialog `Your mysterious passenger has spoken hardly a word during the entire trip. As soon as you land, he hands you your payment of <payment> and disappears into a crowd, off to do who-knows-what.`
+
+
+
+mission "Shady passenger transport 2"
+	name "Shady passenger transport"
+	description "Discreetly transport two passengers to <destination>. If you are caught with them, you may be fined."
+	minor
+	repeat
+	source
+		attributes "near earth" core urban
+	destination
+		attributes "frontier" "dirt belt" "rim" "north" "south" "pirate"
+		distance 3 7
+	passengers 2
+	illegal 150000 `Your passengers turn out to be wanted criminals. As they are hauled off in chains, you vehemently protest that you knew nothing of their shady histories. The authorities threaten you with jail time, and you quickly contact a lawyer. You wind up quite a bit poorer through a combination of fines, bribes, and attorney's fees (the breakdown remains somewhat unclear).`
+		stealth
+
+	to offer
+		random < 8
+		"rejected illegal jobs" < 3
+		"getaway driver" > 2
+
+	on offer
+		dialog `As you are walking through the spaceport, two men in nondescript clothing appear out of nowhere and offer <payment> for discreet, no-questions-asked passage to <destination>.`
+
+	on accept
+		"rejected illegal jobs" --
+		"getaway driver" ++
+		dialog `You quickly bundle the men onto your ship before anyone sees them. As you're helping unpack their luggage, one of them casually mentions that they may have attracted the attention of some bounty hunters, and would prefer not to be captured. You should probably avoid being scanned, too."`
+		"rejected illegal jobs" --
```

Comment:
Rejected illegal jobs is listed here twice.

Reply:
Whoops, not intentional. Fixed.